Resources/Prototypes/Entities/Objects/Weapons/revolutionaries.yml

```diff
@@ -0,0 +1,43 @@
+- type: entity
+  name: Hypnotic flash
+  parent: Flash
+  id: RevolutionaryFlash
+  description: Acts like a standard flash, having more uses at the cost of having a weaker stun. If used on a person not wearing eye protection and not apart of command or security it will convert them over to the revolution
+  components:
+    - type: Sprite
+      sprite: Objects/Weapons/Melee/revoflash.rsi
+      state: flash
+    - type: Flash
+      duration: 1000
+      range: 1
+      aoeFlashDuration: 500
+      uses: 10
+    - type: Item
+      size: 5
+      sprite: Objects/Weapons/Melee/revoflash.rsi
+
+- type: entity
+  name: Revolution flag
+  parent: BaseItem
+  id: RevolutionFlag
+  description: A flag that bears the emblem of the revolution, it may not be much but it may bring courage to those that fight for freedom
+  components:
+    - type: Sprite
+      sprite: Objects/Weapons/Melee/revolutionflag.rsi
+      state: icon
```

Comment:
netsync false

Reply:
Could you elaborate on this one?

Reply:
sprites shouldn't be netsynced and the default for components is that they're always netsynced so you need to turn it off manually via `netsync: false`

Reply:
Ah I see